packages/vite-plugin/src/index.ts

```diff
@@ -65,11 +69,15 @@ export function vanillaExtractPlugin({
         postCssConfig = await resolvePostcssConfig(config);
       }
 
-      virtualExt = `.vanilla.${config.command === 'serve' ? 'js' : 'css'}`;
+      virtualExt = config.command === 'serve' ? virtualExtJs : virtualExtCss;
```

Comment:
If `virtualExt` isn't the point of truth for what type of virtual file is loaded anymore then I'd prefer to remove it entirely and make the all the decisions inline.